src/benchmark/operators/sort_benchmark.cpp

```diff
@@ -5,18 +5,45 @@
 #include "../micro_benchmark_basic_fixture.hpp"
 #include "operators/sort.hpp"
 #include "operators/table_wrapper.hpp"
+#include "synthetic_table_generator.hpp"
 
 namespace opossum {
 
-BENCHMARK_F(MicroBenchmarkBasicFixture, BM_Sort)(benchmark::State& state) {
-  _clear_cache();
+class SortBenchmark : public MicroBenchmarkBasicFixture {
+ public:
+  void BM_Sort(benchmark::State& state) {
+    _clear_cache();
 
-  auto warm_up = std::make_shared<Sort>(_table_wrapper_a, ColumnID{0} /* "a" */, OrderByMode::Ascending);
-  warm_up->execute();
-  for (auto _ : state) {
-    auto sort = std::make_shared<Sort>(_table_wrapper_a, ColumnID{0} /* "a" */, OrderByMode::Ascending);
-    sort->execute();
+    auto warm_up = std::make_shared<Sort>(_table_wrapper_a, ColumnID{0} /* "a" */, OrderByMode::Ascending);
+    warm_up->execute();
+    for (auto _ : state) {
+      auto sort = std::make_shared<Sort>(_table_wrapper_a, ColumnID{0} /* "a" */, OrderByMode::Ascending);
+      sort->execute();
+    }
   }
-}
+
+ protected:
+  void SetUpWithOverriddenSize(const ChunkOffset& chunk_size, const size_t& row_count) {
```

Comment:
There is nothing wrong about this line, just two small proposals for improvement from my side:
1) The primitive types chunk_size and row_count may be referred to by value.
2) You already align parameter names with the methods that you use inside this method. Why do you not also align the parameter order, i.e., switch chunk_size and row_count?

Reply:
Thanks! Implemented in f8fb9c739fb038f91b82e041248d5b83eb5537e5